server/utils/tasks/index.ts

```diff
@@ -21,4 +25,36 @@ const groupByAllocation = (tasks: Array<Task>) => {
 
   return result
 }
-export { groupByAllocation, allocatedTableRows, unallocatedTableRows }
+
+const applicationSummary = (application: Application): Array<SummaryListItem> => {
```

Comment:
Might be useful to get who is currently allocated, as we do have that information (via `allocatedToStaffMember`)

Reply:
Oh, course, we're fetching the application at the point aren't we? D'oh!

Reply:
Yeah, it's a good idea though.  
I feel like it's something the user would want/need to see at this point